server/routes/schools.js

```diff
@@ -0,0 +1,67 @@
+var express = require('express');
+var router = express.Router();
+var bodyParser = require("body-parser");
+
+const { PrismaClient } = require('@prisma/client')
+
+router.use(bodyParser.json());
+router.use(bodyParser.urlencoded({ extended: false }));
+
+async function getSchools(){
+  const allMySchools = await prisma2.school.findMany();
+  return allMySchools
+}
+
+async function postSchools(body){
+  let name = body.name;
+  let address = body.address;
+  let numberOfStudents = body.numberOfStudents;
+  let specialityPrograms = body.specialityPrograms;
+
+  if(!specialityPrograms || !name || !address || !numberOfStudents ){
+    return 422;
+  }
+ //make sure you have all the parameters
+  if(numberOfStudents < 0){
+   
+    return 422;
+  }
+
+  let test = await prisma2.school.create({data:{
+      name: body.name,
+      address:body.address,
+      numberOfStudents: body.numberOfStudents,
+      specialityPrograms: body.specialityPrograms
+  }})
+  return test;
+}
+
+async function deleteSchools(body){
+  let response = await prisma2.school.delete({
+    where:{
+      id: body.id
+    }
+  })
+
+  return response
+}
+
+const prisma2 = new PrismaClient()
+/* GET users listing. */
+router.get('/',  async function(req,res,next){
+  res.send(getSchools());
```

Comment:
I think this needs to be `res.send(await getSchools());` so that Node waits for the async `getSchools()` function to finish evaluating before passing the value into `res.send`. It may work locally now but create issues down the line, unless there's some example from ExpressJS docs that explicitly allow this